diagnostics/ENSO_RWS/input_data/obs_data/ENSO_RWS/enso_rws

```diff
@@ -0,0 +1 @@
+
```

Comment:
remove empty file, or add contents back to file if it should be included

Reply:
deleted